src/meili/map_matching.cc

```diff
@@ -46,11 +51,12 @@ State::route(const std::vector<const State*>& states,
   }
 
   // Route
-  labelset_ = std::make_shared<LabelSet>(std::ceil(max_route_distance));
+  labelset_ = std::make_shared<LabelSet>(max_route_distance);
```

Comment:
Sorry false alarm. It should be `max_route_distance`, because the sort cost in the queue is always distance.